ext/drreg/drreg.c

```diff
@@ -1828,8 +2704,16 @@ tls_data_init(per_thread_t *pt)
         drvector_init(&pt->reg[GPR_IDX(reg)].live, 20, false /*!synch*/, NULL);
         pt->reg[GPR_IDX(reg)].native = true;
     }
+    for (reg = DR_REG_APPLICABLE_START_SIMD; reg <= DR_REG_APPLICABLE_STOP_SIMD; reg++) {
+        drvector_init(&pt->simd_reg[SIMD_IDX(reg)].live, MCXT_NUM_SIMD_SLOTS + 10,
```

Comment:
Where is the "10" coming from?

Reply:
This again was to init a larger than needed vector, after seeing how vectors for GPRs are done (which has a starting size of 20).

Reply:
Why do you need more than MCXT_NUM_SIMD_SLOTS? This is always the max, e.g. mcontext is using it to size its structures

Reply:
Done.